contrib/collectd/template.py

```diff
@@ -0,0 +1,52 @@
+pkgname = "collectd"
+pkgver = "5.12.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+        "--prefix=/usr/bin",
+        "--disable-pcie_errors",
```

Comment:
i see you disabled this because otherwise it fails in tests with sigill, but you should fix that instead of disabling it

Reply:
here

```
the test intentionally breaks fortify and expects a call to pcie_read with
overflowing size to fail, but that just crashes in fortify instead of returning
an error
--
--- a/Makefile.am
+++ b/Makefile.am
@@ -1692,7 +1692,7 @@
 	src/daemon/utils_llist.c \
 	src/daemon/configfile.c \
 	src/daemon/types_list.c
-test_plugin_pcie_errors_CPPFLAGS = $(AM_CPPFLAGS)
+test_plugin_pcie_errors_CPPFLAGS = $(AM_CPPFLAGS) -U_FORTIFY_SOURCE
 test_plugin_pcie_errors_LDFLAGS = $(PLUGIN_LDFLAGS)
 test_plugin_pcie_errors_LDADD = liboconfig.la libplugin_mock.la
 check_PROGRAMS += test_plugin_pcie_errors

```

Reply:
added